translations/pt.json

```diff
@@ -0,0 +1,223 @@
+        "The official Tarkov.dev Discord bot! An open source project by The Hideout to help you play Escape from Tarkov.": "O bot para o Discord oficial de Tarkov.dev! Um projeto de código aberto por The Hideout, feito para te ajudar a jogar Escape de Tarkov.",
+        "The token you provided is invalid. Provide your [TarkovTracker API token](https://tarkovtracker.io/settings/) to link your account.": "The token providenciado é inválido. Providencie sua [TarkovTracker API token](https:\/\/tarkovtracker.io\/settings\/) para conectar à sua conta.",
+        "There was an error processing your request.": "Ocorreu um erro no processamento de seu pedido.",
+        "These values are used to determine craft/barter/price unlocks and flea market fees.": "Esses valores são usados para determinar fabricações\/trocas\/preços desbloqueadas e as taxas do mercado de pulgas",
+        "Time": "Tempo",
+        "Too many notes to display.": "Notas demais para mostrar.",
+        "Total": "Total",
+        "Trader restocks": "Renovação de estoque do comerciante",
+        "Traders": "Comerciantes",
+        "Unknown": "Desconhecido",
+        "Value": "Valor",
+        "Velo": "Velocidade",
+        "Want to check the status of our services (api, website, bot, etc)?": "Quer checar a situação de nossos serviços (api, website, bot, etc)?",
+        "Want to contribute to the bot or checkout the source code? View the project on GitHub!": "Quer contribuir para o bot ou checar o código fonte? Veja o projeto no GitHub!",
+        "Weight": "Peso",
+        "Where [command] is one of the following commands:": "Onde [command] é um dos seguintes comandos:",
+        "You can learn more about the bot's commands by entering:": "Você pode aprender mais sobre os comandos do bot digitando:",
+        "You do not have any saved progress. Below are the defaults used to determine craft/barter/price unlocks and flea market fees.": "Você não tem nenhum progresso salvo. Abaixo estão os padrões usados para determinar fabricações\/trocas\/preços desbloqueadas e as taxas do mercado de pulgas.",
+        "You have restock alerts enabled for": "Você tem alertas de renovação de estoque habilitadas para",
+        "You must be an administrator to set channel restock alerts.": "É necessário ser administrador para definir canais de alerta de renovação de estoque.",
+        "You must supply your [TarkovTracker API token](https://tarkovtracker.io/settings/) to link your account.": "Você deve providenciar seu [Token de API da TarkovTracker](https:\/\/tarkovtracker.io\/settings\/) para conectar com a sua conta.",
+        "You need to specify a search term": "Você precisa especificar um termo de procura",
+        "You need to specify an ammo type": "É necessário especificar um tipo de munição",
+        "Your hideout progress will update from TarkovTracker {{updateTime}}.": "O progresso do seu esconderijo vai ser atualizado pela TarkovTracker {{updateTime}}."
+    },
+    "command": {
```

Comment:
Thanks for contributing a pt-BR translation!

In the "command" namespace, anything that doesn't end in "_desc" is a command name or command argument name. That means they can't contain spaces and should be in all lowercase letters. They should also generally be kept shorter because the user may have to type them out, and it's nice to save the user from lots of typing.

Reply:
ok, so one question: in the ones with "_desc", do I keep the description, or do I change it to be like this:

Reply:
For comparison' sake, the current `"ammo_desc"` is `"ammo_desc": "obter informações sobre munições"`

Reply:
In the command namespace, anything that ends in "_desc" can have spaces and full punctuation. So the example is perfectly fine!